test/integration/recursor_source/serverspec/recursor_source_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'spec_helper'
+
+describe 'A PowerDNS Resolver' do
+  it 'is running' do
+    expect(process('pdns_recursor')).to be_running
+  end
+
+  it 'can resolve example.com' do
+    expect(command('dig @localhost example.com').stdout).to \
```

Comment:
For this to be a realistic test you'll want to also use the resolve cookbook to swap the default stub resolver with `127.0.0.1`